clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -664,6 +664,46 @@ describe("Fides-js TCF", () => {
             vendorsDisclosed
           );
         });
+        // verify the data layer variables
+        cy.get("@dataLayerPush")
+          .should("have.been.calledThrice")
+          // First call should be from initialization, before the user accepts all
+          .its("firstCall.args.0")
+          .should("deep.equal", {
+            event: "FidesInitialized",
+            Fides: {
+              consent: {},
+              extraDetails: undefined,
+              fides_string: undefined,
+            },
+          });
+        // FidesUpdating call
+        cy.get("@dataLayerPush")
+          .its("secondCall.args.0.Fides")
+          .should("deep.include", {
+            consent: {},
+            extraDetails: {
+              consentMethod: "accept",
+            },
+          });
+        cy.get("@dataLayerPush")
+          .its("secondCall.args.0")
+          .its("Fides.fides_string")
+          .should("contain", ",1~");
```

Comment:
partial match and not as part of the assertion on line 681 above, because the `fides_string` generated is inconsistent